acala/scripts/publish-manifest.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+for file in ./project*.yaml; do
+    echo "Publishing $file..."
+    npx subql publish -f "$file"
+done
```

Comment:
If any of these fail then it wont be caught, `set -e` should fix that

```suggestion
#!/bin/bash

set -e

for file in ./project*.yaml; do
    echo "Publishing $file..."
    npx subql publish -f "$file"
done
```

Reply:
Why not just have a top level script to do all of this rather than one for each project?